doc-Deployment_Planning_Guide/topics/planning.adoc

```diff
@@ -212,9 +212,48 @@ Providers can be added at any level of tenancy. Once added, a provider is visibl
 [[load-balancer]]
 === Using a Load Balancer
 
-Deploying several {product-title_short} appliances with a load balancer can be useful for performance.
+Deploying multiple user interface worker appliances and placing them behind a third-party load balancer allows for redundancy and improved performance. This requires extra configuration in both the load balancer and in the {product-title_short} user interface worker appliances.
 
-However, running several appliances behind a load balancer can cause login issues from the Self Service user interface because `memcached`, the service storing the session in memory, is not shared between the appliances. To prevent this, configure your environment using the steps below to run the `memcached` service on one appliance and have it shared with the other appliances.
+.Load balancer configuration
+
+* Configure the load balancer to use sticky sessions. This ensures that when a session is started, all requests for that session are sent to the same worker appliance.
+* Configure the load balancer to test for connectivity using the {product-title_short} ping response page: `https://appliance_name/ping`. The expected reply from the appliance is the text string _pong_. Using this URL is preferable to the appliance login URL as it does not establish a connection to the database. 
+
+.Appliance configuration
+
+By default, {product-title_short} user interface worker appliances store session data in the local appliance’s memcache. When operating behind a load balancer, it is better to store the user session data in the shared database (VMDB) rather than in memcache, which is not shared. 
+
+When using a load balacner, configure appliances that have the *User Interface* role enabled to store session data in the database. As a result, the user does not need to re-login if the load balancer redirects them to an alternative server in case the original user interface worker is unresponsive. 
```

Comment:
If you remove the first section, I'd just revide 'database' to something like "shared database (VMDB)"